pyscriptjs/examples/logic.py

```diff
@@ -0,0 +1,150 @@
+from js import console, document
+
+answer = Element("typing-text")
+num_first = ""
+num_second = ""
+sign_click = ""
+cal_type = ""
+numbers = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "."]
+sign = ["equal-to", "reverse", "ac", "percent", "plus-or-minus"]
+
+
+def numbers_clicked(args):
+    global num_first
+    global num_second
+    global sign_click
+    typed_in = args.target.innerText
+    console.log(args.target.innerText, args)
+    if typed_in in numbers and sign_click == "":
+        answer.element.innerText += typed_in
+        num_first = answer.element.innerText
+        console.log(answer.element.innerText, num_first)
+    elif typed_in in numbers and sign_click != "":
+        document.getElementById("typing-text").innerHTML += typed_in
+        num_second += typed_in
+        console.log(answer.element.innerText, num_second, typed_in)
+
+
+def sign_clicked(args):
+    global num_first
+    global num_second
+    global sign_click
+    global cal_type
+    console.log(args.target.innerText, args)
+    if args.target.id == "equal-to":
+        console.log("give total")
+        calculate()
+    elif args.target.id == "reverse":
+        console.log("give reverse")
+        clear_all()
+    elif args.target.id == "ac":
+        console.log("give ac")
+        clear_all()
+    elif args.target.id == "percent" and sign_click == "" and num_second == "":
+        console.log("give percent")
+        num_first = int(num_first) / 100
+        document.getElementById("typing-text").innerText = num_first
+    elif (
+        args.target.id == "plus-or-minus"
+        and num_first != ""
+        and sign_click == ""
+        and num_second == ""
+    ):
+        current = document.getElementById("typing-text").innerText
```

Comment:
This could be `current = Element("typing-text").element.innerText`

Reply:
same comment applies to many other cases below

Reply:
Okay, will get that fixed.